### PR TITLE
Fix upsampling test case on ppc

### PR DESCRIPTION
**Pull request by charleshofer**
Power and x86 are giving slightly different results when scaling images up using `torch.nn.functional.interpolate` and when using OpenCV's `resize`. This is causing `test_upsampling_not_recompute_scale_factor` to fail on Power, but not x86. This changes the expected value to what OpenCV on Power produces if the test case is running on Power as well.

See https://github.com/pytorch/pytorch/issues/31915

@ezyang

### CODE REVIEW

**Review by ezyang — APPROVED** [review]

**Comment by ezyang** [comment]
TBH, I was expecting you to crank down the precision, but this is ok too

**Comment by ezyang** [comment]
waiting on cI

**Comment by kostmo** [comment]
## :pill: CircleCI build failures summary and remediations
As of commit 99ea98e4:


**None of the build failures appear to be your fault**.

* **2/2** [broken upstream](https://dr.pytorch.org/master-timeline.html?count=1&sha1=2471ddc96c56162861f41f3b6b772b4b4e773ea7&should_suppress_scheduled_builds=true&should_suppress_fully_successful_columns=true&max_columns_suppress_successful=35) at merge base 2471ddc9 since Jan 29
  <details>
  <summary>
  Please rebase on the <code>viable/strict</code> branch (expand for instructions)
  </summary>
  
  If your commit is newer than `viable/strict`, you can try basing on an older, stable commit:
  ```
  git fetch origin viable/strict
  git rebase --onto viable/strict $(git merge-base origin/master HEAD)
  ```
  If your commit is older than `viable/strict`:
  ```
  git fetch origin viable/strict
  git rebase viable/strict
  ```
  Check out the [recency history](https://dr.pytorch.org/master-viable-commits.html) of this "viable master" tracking branch.
  
  </details>
  


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=99ea98e419e09db7849bdf017a260cdbe5df757d).

### :construction: 2 upstream failures recognized by patterns:
These builds matched patterns, but were probably **caused by upstream breakages**:
* [caffe2_onnx_py3_6_clang7_ubuntu16_04_test](https://dr.pytorch.org/build-details.html?build_id=92792322) from Jan 29
* [caffe2_onnx_py2_gcc5_ubuntu16_04_test](https://dr.pytorch.org/build-details.html?build_id=92792491) from Jan 29



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

**Comment by charleshofer** [comment]
Yeah, I thought this might be a better solution than turning down the precision

**Comment by charleshofer** [comment]
@ezyang That pending CI check looks like it's complete, but the label in the top left says it's "Queued" still? Not too familiar with circleci though, so maybe I'm just not understanding the output.

It looks like the two failures aren't related to my change.
